### ISSUE TITLE
Review the sphinx-book-theme  for a11y issues that are too large to deal with

### ISSUE THREAD

**Issue by feanil** [openedx/docs.openedx.org#230]
If we find major issues, we should work to fix them upstream or consider moving to a more accessible theme with a strong preference for fixing up-stream over maintaining our own theme.

**Comment by feanil** [comment]
There weren't too many issues but one that was on the main page was a contrast issue with the "Home" buttons.

I tried to override the primary color for the design theme to deal with this but there seems to be a bug where our custom.css file loads earlier than the design-style.css file that the extension provides.

I made https://github.com/executablebooks/sphinx-design/pull/124 upstream to just fix the upstream theme and I'm gonna spend a little bit more time to see if I can figure out why the CSS is loading in the wrong order.
